### DISCUSSION TITLE
Add an encryptor

### DISCUSSION THREAD

**Discussion by gamezaSRC** [bedrock-tool/bedrocktool#22] [Ideas]
You could add an encryptor for our resource packs, that will be very useful

**Comment by olebeck** [comment]
https://encryptmypack.com/ there's an existing website for this, may add it later if I have a use for it

**Reply by gamezaSRC** [reply]
Yea i know that there is a website for that but the API it's broken

**Reply by gamezaSRC** [reply]
And that website is always down
